### ISSUE TITLE
LoadFasterWhisperModel node is missing

### ISSUE THREAD

**Issue by naumanahmed19** [jhj0517/ComfyUI-faster-whisper#4]
It seems LoadFasterWhisperModel node is missing can you please check

![image](https://github.com/user-attachments/assets/132b8e8a-d765-4fb5-b1b6-26f1a32748e6)

**Comment by jhj0517** [comment]
Hi. Can you check if the node is installed properly?
You can check if the `faster-whisper` is installed with
```
pip show faster-whisper
```

And if you're using the portable version of the Comfy, you should've done:
```
python_embeded\python.exe -m pip install -r ComfyUI\custom_nodes\ComfyUI-faster-whisper\requirements.txt
```

**Comment by naumanahmed19** [comment]
here is the output


```C:\Users\xxxxx\ComfyUI>pip show faster-whisper
Name: faster-whisper
Version: 1.0.3
Summary: Faster Whisper transcription with CTranslate2
Home-page: https://github.com/SYSTRAN/faster-whisper
Author: Guillaume Klein
Author-email:
License: MIT
Location: C:\Users\Saali\AppData\Local\Programs\Python\Python312\Lib\site-packages
Requires: av, ctranslate2, huggingface-hub, onnxruntime, tokenizers
Required-by:
```

all other nodes are loading just fine

![image](https://github.com/user-attachments/assets/c2bfa119-e645-4554-a253-0aaa56c693d0)

**Comment by jhj0517** [comment]
OK, I just tested with the latest version of Comfy and reproduced the same bug. Something custom node syntax might have been changed.

**Comment by jhj0517** [comment]
@naumanahmed19 I found a mistake and fixed in #5.
Latest version should work fine!

**Comment by jhj0517** [comment]
Closing since this is fixed, 
Please feel free to re open if you still have the problem!
